bundle-size/webhooks.js

```diff
@@ -93,12 +93,26 @@ exports.installGitHubWebhooks = (app, db, githubUtils) => {
       return;
     }
 
+    const isBundleSizeSuperApprover = await githubUtils.isBundleSizeSuperApprover(
```

Comment:
Nit: `BundleSize` is obvious here. If it were just `isSuperApprover` it would fit on one line :laughing:

Reply:
Done.